hazelcast/src/main/java/com/hazelcast/sql/impl/SqlServiceImpl.java

```diff
@@ -351,4 +362,11 @@ public void closeOnError(QueryId queryId) {
         }
         getInternalService().getClientStateRegistry().closeOnError(queryId);
     }
+
+    /**
+     * Added for testing
+     */
+    void setOptimizer(SqlOptimizer optimizer) {
```

Comment:
not needed if we test PH instead of this class

Reply:
removed